packages/driver/src/cy/commands/navigation.ts

```diff
@@ -422,12 +422,12 @@ const stabilityChanged = (Cypress, state, config, stable) => {
 
       const onInternalWindowLoad = (details) => {
         switch (details.type) {
-          case 'same:domain':
+          case 'same:origin':
             return onWindowLoad(details.window)
-          case 'cross:domain':
-            return onCrossDomainWindowLoad(details)
-          case 'cross:domain:failure':
-            return onCrossDomainFailure(details.error)
+          case 'cross:origin':
+            return onCrossOriginWindowLoad(details)
+          case 'cross:origin:failure':
+            return onCrossOriginFailure(details.error)
           default:
             throw new Error(`Unexpected internal:window:load type: ${details?.type}`)
```

Comment:
not related, how is this handles/displayed to the user handle this if this was hit?

Reply:
I believe that cy.fail would eventually catch the error and fail the current command